test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -361,6 +367,59 @@ function primitive_to_prognostic!(
 
 end
 
+function compute_filter_argument!(
+    target::IntrinsicEDMFFilter,
+    filter_state::Vars,
+    state::Vars,
+    aux::Vars,
+)
+    gm = state
+    up = state.turbconv.updraft
+    en = state.turbconv.environment
+    fs_up = filter_state.turbconv.updraft
+    fs_en = filter_state.turbconv.environment
+    ρ_inv = 1 / gm.ρ
+
+    ft_en = target.environment
```

Comment:
```suggestion
    ft_en = filter_state.turbconv.environment
```
+ similar changes below